### PR TITLE
Remove CoursePlus references

### PR DESCRIPTION
**Pull request by ehumph**
This addresses issue #5 

All references to CoursePlus (for both dropboxes and surveys) were deleted.

I did this by searching for "CoursePlus" in all the relevant files. References were deleted from the homework Rmds, the syllabus and schedules, the email to students, and the modules.

### CODE REVIEW

**Comment by ehumph** [comment]
In addition to removing the CoursePlus links for surveys and dropboxes in the config.yml, I had to comment out the link to the google form and slack workspace in order to prevent a token error when attempted to render the website. These will need to be changed to the proper locations when they are ready.

**Comment by ehumph** [comment]
I also changed the language about how to submit the homework (at the beginning of each homework Rmd) to:

"Homework is optional, but we recommend it so you can get the most out of this course."

**Comment by ehumph** [comment]
> In addition to removing the CoursePlus links for surveys and dropboxes in the config.yml, I had to comment out the link to the google form and slack workspace in order to prevent a token error when attempted to render the website. These will need to be changed to the proper locations when they are ready.

Just kidding. Turns out it was a misplaced space causing the actual issue. The google form and slack workspace links are no longer commented out, although they will need to be changed to the proper addresses when ready.

File: modules/Project_Guidelines/Project_Guidelines.Rmd [hunk 1]
```diff
@@ -10,7 +10,7 @@ This project is supposed to get you used to working with R in a way that would b
 1) Please identify a dataset to analyze. Any dataset will do, as long as you can perform the following requirements and you **do not violate any privacy restrictions** for the data. 
 Also please ensure that the instructors have access to your data so that they can evaluate your project. 
 
-**Thus send us a link to your data if it is hosted somewhere or make sure to turn in a data file (csv, excel etc.) on CoursePlus.** If the data is not hosted publicly somewhere, you must turn it in on Drop Box with your other files. 
+**Thus send us a link to your data if it is hosted somewhere ** If the data is not hosted publicly somewhere, you must turn it in on Drop Box with your other files.
```
Comment (avahoffman) [inline modules/Project_Guidelines/Project_Guidelines.Rmd:13]: Drop Box is a JHU thing - I think this can just take this chunk out

**Pull request merged**